companion/lib/Surface/USB/Loupedeck.ts

```diff
@@ -50,6 +50,7 @@ export class SurfaceUSBLoupedeck extends EventEmitter<SurfacePanelEvents> implem
 	readonly #writeQueue: ImageWriteQueue<string, [ImageResult]>
 
 	config: Record<string, any>
+	DisplayColors: Record<string, any>
```

Comment:
It would be good to have stronger types for this object, to minimise the risk of type errors in future refactorings